File: app/src/main/java/ru/example/newsreader/retrofit/HabrService.java
package ru.example.newsreader.retrofit;

import retrofit2.Call;
import retrofit2.http.GET;
import ru.example.newsreader.models.RSSFeed;

public interface HabrService {
    @GET("rss/hubs/all/")
    Call<RSSFeed> getArticles();
}

File: app/src/main/java/ru/example/newsreader/retrofit/HabrService.kt
package ru.example.newsreader.retrofit


import io.reactivex.Observable
import retrofit2.http.GET
import ru.example.newsreader.models.RSSFeed

interface HabrService {
    @GET("rss/hubs/all/")
    fun getArticles(): Observable<RSSFeed>
}